src/MongoDB/Session.c

```diff
@@ -237,6 +385,10 @@ ZEND_BEGIN_ARG_INFO_EX(ai_Session_advanceOperationTime, 0, 0, 1)
 	ZEND_ARG_INFO(0, timestamp)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_INFO_EX(ai_Session_startTransAction, 0, 0, 0)
```

Comment:
I don't think the "A" should be capitalized: "startTransaction"

Reply:
Good catch.